src/styles/header-block.scss

```diff
@@ -0,0 +1,77 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  .logo {
+    display: flex;
+    justify-content: center;
+    height: 40px;
+    width: 40px;
+    margin: 10px 50px;
+  }
+
+  .nav {
+    margin-right: 50px;
+
+    &__list {
+      display: flex;
+      flex-direction: row;
+      margin: 0;
+      padding: 0;
+
+      .nav__item {
+        display: block;
+        position: relative;
+        margin: 0 10px;
+
+        .nav__link {
```

Comment:
why these styles are nested?
make every nested only inside `nav` and use `&` SCSS nesting